build/xtask/src/dist.rs

```diff
@@ -2113,7 +2121,7 @@ fn write_srec(
         panic!("SREC limit of 2^24 output sections exceeded");
     }
 
-    srec_out.push(srec::Record::S7(srec::Address32(kentry)));
+    srec_out.push(srec::Record::S7(srec::Address32(kentry as u32)));
```

Comment:
Is there anything to check that the `kentry` will fit in a `u32`? If not, an error should be generated instead of silently ignoring that.

Reply:
Fixed in next patchset.